### PR TITLE
Fixed wrong return type

### PR DESCRIPTION
**Pull request by AchimTuran**
Fixed wrong return type in libXBMC_addon and improved code formatting.

### CODE REVIEW

File: xbmc/utils/CPUInfo.h [hunk 1]
```diff
@@ -102,7 +102,7 @@ class CCPUInfo
 
 private:
   bool readProcStat(unsigned long long& user, unsigned long long& nice, unsigned long long& system,
-    unsigned long long& idle, unsigned long long& io);
+                    unsigned long long& idle, unsigned long long& io);
```
Comment (FernetMenta) [inline xbmc/utils/CPUInfo.h]: is this intended?

Comment (AchimTuran) [inline xbmc/utils/CPUInfo.h]: Yes I thought it's a better coding style.

Comment (FernetMenta) [inline xbmc/utils/CPUInfo.h]: ahh, now I see that it belongs to the function. fine

Comment (AchimTuran) [inline xbmc/utils/CPUInfo.h]: Yes correct, github's view for this is not very pretty.

Comment (xhaggi) [inline xbmc/utils/CPUInfo.h]: but this cosmetic correction is not related to this PR so please remove it.

Comment (AchimTuran) [inline xbmc/utils/CPUInfo.h]: I put it in https://github.com/xbmc/xbmc/pull/6395

Comment (AchimTuran) [inline xbmc/utils/CPUInfo.h]: Now this is splitted in two commits. I hope it is ok now.

**Comment by AchimTuran** [comment]
Now this is splitted in two commits. I hope it is ok now.

**Comment by FernetMenta** [comment]
jenkins build and merge
